apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V69__new_schema_hospice_claims_migration.sql

```diff
@@ -0,0 +1,528 @@
+-- NEW_SCHEMA_HOSPICE_CLAIMS.SQL
+-- flyway migration for HOSPICE_CLAIMS and HOSPICE_CLAIM_LINES tables into
+-- a new schema structure that:
+--   1) changes data type of CLM_ID, BENE_ID from varchar to BIGINT.
+--   2) change data type of CLM_GROUP_ID from numeric to BIGINT
+--   3) organizes parent claim table (HOSPICE_CLAIMS_NEW) such that common
+--      claims data columns are organized at top of table structure.
+--
+--      The following db columns were redefined from NUMERIC to more
+--      appropriate data type(s):
+--
+--      from: clm_utlztn_day_cnt numeric         : changed to integer
+--      from: bene_hospc_prd_cnt numeric(2,0)    : changed to smallint
+--      from: clm_line_num numeric               : changed to smallint
+--      from rev_cntr_unit_cnt numeric           : changed to integer
+--
+--       Current db data values for changed columns
+--      +-----------------------+--------+------+-------+
+--      |  db Column            |   Max  |  Min | Scale |
+--      +-----------------------+--------+------+-------+
+--      | CLM_UTLZTN_DAY_CNT    |    240 | -87  |   0   |
+--      | BENE_HOSPC_PRD_CNT    |      3 |   0  |   0   |
+--      | CLM_LINE_NUM          |    450 |   1  |   0   |
+--      | REV_CNTR_UNIT_CNT     | 904404 |   0  |   0   |
+--      +-----------------------+--------+------+-------+   
```

Comment:
apart from clm_line_num, I think the additional scope is a risk that we should not take right now